cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -170,8 +170,14 @@ func generateClusterConfig(clusterName string) error {
 			}
 			datacenterYaml = dcyaml
 
-			cpMachineConfig := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName + "-cp")
-			workerMachineConfig := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName)
+			cpMachineConfig, err := v1alpha1.NewTinkerbellMachineConfigGenerate(clusterName + "-cp")
```

Comment:
The `clusterName + "-cp"` really ought to be in a function somewhere, probably not in this package, so it can be recreated as needed (testing etc).

Reply:
Do you recommend which package to put this in?

Reply:
Having skimmed the relevant code, the function would be relevant to all providers so `/pkg/providers` seems appropriate. I would then substitute the vsphere naming with your new function also :)

Reply:
Yeah and I'll create one for etcd as well